specification/dimensions/pricingcategory.md

```diff
@@ -0,0 +1,47 @@
+# Pricing Category
+
+Pricing Category describes the kind of pricing model used for a charge at the time it was used or purchased. Pricing Category is commonly used for scenarios like differentiating between charges at on-demand vs. reduced prices when investigating optimization opportunities, like for commitment-based discount coverage.
+
+PricingCategory MUST follow the requirements listed below:
+
+* PricingCategory MUST be present in the billing data and MUST be of type String.
+* PricingCategory MUST be null if SkuPriceId is null and MUST NOT be null if SkuPriceId is not null.
+* PricingCategory MUST be one of the allowed values.
+* PricingCategory MUST be "On-Demand" when pricing is predetermined at the standard rate for the billing account.
+* PricingCategory MUST be "Commitment-Based" when CommitmentDiscountId is not null.
+* PricingCategory MUST be "Dynamic" when pricing is determined by the provider and may change over time, regardless of predetermined agreement pricing.
+* PricingCategory MUST be "Other" when there is a pricing model but none of the current allowed values apply.
+
+## Column ID
+
+PricingCategory
+
+## Display name
+
+Pricing Category
+
+## Description
+
+Describes the kind of pricing model used for a charge at the time it was used or purchased.
+
+## Content constraints
+
+| Constraint      | Value          |
+| :-------------- | :------------- |
+| Column required | True           |
+| Data type       | String         |
+| Allows nulls    | True           |
+| Value format    | list-of-values |
+
+Allowed values:
+
+| Value            | Description                                                                                                                                                                                                  |
+| :--------------- | :----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| On-Demand        | Charges priced at the standard rate for the billing account. This includes any applicable negotiated discounts and volume/tiered pricing, but does not include dynamic or commitment-based discount pricing. |
+| Dynamic          | Charges priced at a variable rate determined by the provider. This includes any product or service with a unit price the provider can change without notice, like interruptible or low priority resources.   |
+| Commitment-Based | Charges with reduced prices due to a commitment-based discount specified by the Commitment Discount ID.                                                                                                      |
```

Comment:
I covered this in the supporting content, but the concern with Discounted is that it groups too many different types of pricing models together. There's a big difference between a commitment-based discount and a flat rate discount and other discounts that will come in the future. If we group them all together, then it dilutes the goal of differentiating the categories of discounted pricing approaches that the column is seeking.

Reply:
I was not quite sure how to put my answer into words, but I think I agree with the way @flanakin described it - in this column we are interested in the _manner_ in which it was priced (and in this case it was a discount based on a commitment).

Reply:
Resolving this so we can close this out.

Reply:
Hi team, if the value is 'Commitment-Based' are we making sure in other column there is a arn to id the commitment, I'm saying this because I have an open conversation with our AWS TAM, since the purchase option data from AWS CUR is not enough so we need to add a CASE to validate that if it has a commitment arn it should be mark as 'Commitment-Based'